### ISSUE TITLE
Reproduce resutls on Caltech dataset

### ISSUE THREAD

**Issue by dogdogpee** [hasanirtiza/Pedestron#153]
Claim: Cascade Mask R-CNN, Caltech dataset, 1.7%, 25%, 14 epoch

Reproduce: following configs/elephant/caltech)/cascade_hrnet.py/ got 10%. 58% on caltech at 14 epoch.

Question: what hyperparameters do you use to produce the claimed results?

Many thanks!

**Comment by hasanirtiza** [comment]
Refer to section 7 of our [paper](https://arxiv.org/pdf/2201.03176.pdf). It is about progressive finetuning and how we obtained this particular result. The hyperparameters are the one provided in the config file that you are using.

**Comment by dogdogpee** [comment]
Thanks for your reply!

Could you please share the training config of the Cascade R-CNN (HR Backbone) in Table 5 under the case of  Caltech-->Caltech in the paper you mentioned?

Thanks for your time!

**Comment by hasanirtiza** [comment]
Did you try with the config provided in the repo ? I mean did you evaluate every checkpoint ? I do not exactly remember if it was the same config for table 5 and unfortunately I do not have the access to old files anymore (not working there anymore). Nonetheless, what I can suggest is, first start with the config provided in the repo, and in case required, just adjust the learning rate.
